### ISSUE TITLE
Tile indices for man and sou are wrong

### ISSUE THREAD

**Issue by Bevinsky** [NegativeMjark/tenhou-log#7]
The tile indices for man tiles and sou tiles are swapped and do not match those contained in Tenhou logs. Here is an example agari of a round with an All Green (ryuuiisou) yakuman:

`{'type': 'RON', 'player': 0, 'hand': ['6m0', '6m3', '8m0', '8m2', '8m3', 'gd0', 'gd2', 'gd3'], 'fu': 40, 'points': 32000, 'limit': 'yakuman', 'dora': ['2p1'], 'machi': ['
6m3'], 'melds': [{'fromPlayer': 1, 'called': 2, 'type': 'pon', 'tiles': ['4m0', '4m2', '4m3']}, {'fromPlayer': 1, 'called': 0, 'type': 'pon', 'tiles': ['3m0', '3m1', '3m3
']}], 'closed': False, 'fromPlayer': 3, 'yakuman': ['ryuuiisou']}`

Note that the hand contains man tiles and not sou tiles. This is impossible.

**Comment by ChitoKim** [comment]
yeah i guess it would work properly if you swap the suit indices for man and sou. the numbers look like a fine all green hand
